src/main/scala/org/broadinstitute/dsde/workbench/sam/config/GoogleServicesConfig.scala

```diff
@@ -94,12 +95,14 @@ object GoogleServicesConfig {
       config.as[GoogleKeyCacheConfig]("googleKeyCache"),
       config.as[Option[String]]("resourceNamePrefix"),
       config.as[Option[NonEmptyList[ServiceAccountConfig]]]("adminSdkServiceAccounts"),
+      config.as[Option[NonEmptyList[String]]]("adminSdkServiceAccountPaths"),
       config.as[GoogleKmsConfig]("kms"),
       config.getString("terraGoogleOrgNumber")
     )
   }
 }
 
+//TODO: Once ID-91 is merged and deployed, we cann remove ServiceAccountConfig and the adminServiceAccounts variable
```

Comment:
FWIW - when I've had TODO items like this, I will create a ticket for coming back in and removing the old code.  And that way we can make sure ID-91 gets done and we can track the follow-up separately.

Reply:
https://broadworkbench.atlassian.net/browse/ID-97 Exists now!